app/views/orders/_cart.html.erb

```diff
@@ -0,0 +1,50 @@
+<% if current_user.nil? %>
+  <% name = "guest" %>
+<% else %>
+  <% name = current_user.name %>
+<% end %>
+
+<%= link_to "Back to products", products_path %>
+
+<h1>Welcome to your cart, <%= name %>! </h1>
+
+<% if @order.order_items.empty? %>
+  You have nothing in your cart, <%= link_to "Would you like to go back to browse products?", products_path %>
+<% else %>
+
+  <% @order.order_items.each do |item| %>
+  <div class="media" id="cart-list">
+    <% product = Product.find(item.product_id) %>
+
+  <div class="media-left">
+      <%=image_tag product.photo_url, class: "media-object", alt: product.name, id: "cart-pics"%>
+</div>
+  <div class="media-body">
+    <h4 class="media-heading"><%= product.name %>
+    <span class="sub-total">
+      <%=to_money(item.subtotal) %>
+    </span>
+    </h4>
+    <%= to_money(product.price) %>
+    <%= form_for item, :url => product_order_item_path(item.product_id, item.id) do |f| %>
+      <%= f.label :quantity %>
+      <%= f.number_field :quantity, :value => item.quantity %>
```

Comment:
Many entries on places like StackOverflow will use the old hash syntax but it is always good to update to the new syntax whenever you bring solutions over. 

`value: item.quantity`